Core/Debugger/SymbolMap.cpp

```diff
@@ -1003,13 +1003,15 @@ void SymbolMap::GetLabels(std::vector<LabelDefinition> &dest) {
 	if (activeNeedUpdate_)
 		UpdateActiveSymbols();
 
+#ifdef _WIN32
```

Comment:
Do we have to?  This is not Windows only, and is used by the web debugger.  It should work on all platforms.

-[Unknown]

Reply:
Whoops, I first added that in an attempt to get it to compile, then went ahead and implemented for the other platforms. Will remove.